tests/cluster.go

```diff
@@ -534,6 +534,31 @@ func (c *TestCluster) WaitLeader(ops ...WaitOption) string {
 	return ""
 }
 
+// WaitRegionSyncerClientsReady is used to wait the region syncer clients establish the connection.
+// n means wait n clients.
+func (c *TestCluster) WaitRegionSyncerClientsReady(n int) bool {
+	option := &WaitOp{
+		retryTimes:   40,
+		waitInterval: WaitLeaderCheckInterval,
+	}
+	for i := 0; i < option.retryTimes; i++ {
+		name := c.GetLeader()
+		if len(name) == 0 {
+			time.Sleep(option.waitInterval)
+			continue
+		}
+		leaderServer := c.GetServer(name)
+		clus := leaderServer.GetServer().GetRaftCluster()
+		if clus != nil {
+			if len(clus.GetRegionSyncer().GetAllDownstreamNames()) == n {
```

Comment:
Since we only use its length, why bother copying the whole slice?

Reply:
I think it is more clear to return the name of the current downstream than to return only the number, although it is only used in test now.